backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserMutationResolver.java

```diff
@@ -171,11 +171,7 @@ public User updateUserPrivilegesAndGroupAccess(
       @Argument Role role) {
     List<UUID> facilityIdsToAssign = facilities == null ? List.of() : facilities;
     _us.updateUserPrivilegesAndGroupAccess(
-        username,
-        orgExternalId,
-        accessAllFacilities,
-        facilityIdsToAssign,
-        role.toOrganizationRole());
+        username, orgExternalId, accessAllFacilities, facilityIdsToAssign, role);
```

Comment:
I noticed here that an argument is changing from `role.toOrganizationRole` to just `role`. What is the difference between a role vs an organization role? Is the notion of an organization role something that is Okta specific that we will no longer need to maintain with our own user role structure?

Reply:
`OrganizationRole` includes some Okta-specific workarounds (outlined in `OrganizationRole.java`) / supersets `Role` (specified in `Role.java`). Guessing that since we don't want to store the `NO_ACCESS` and `ALL_FACILITIES` org roles (which are Okta structures made to aid the retrieval of all users in an org rather than end-user facing roles), we're switching to storing Roles as we build these into the table itself.

Reply:
Makes sense to me -- thanks Bob!

Reply:
+1 to what Bob said though we are going to be storing ALL_FACILITIES role still. for this method we still convert a Role to an OrgRole, I'm just doing it within the method itself instead to keep it more in line with the way other methods in ApiUserService handle that parameter. it's just a refactor and not a functional change at this point.